set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Localization of rings
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c ~RL RLocal $.
+
+  $( Syntax for ring localization equivalence class operation. $)
+  cerl $a class ~RL $.
+
+  $( Syntax for ring localization operation. $)
+  crloc $a class RLocal $.
+
+  ${
+    $d r s x w a b t $.
+    $( Define the operation giving the equivalence relation used in the
+       localization of a ring by a set.  (Contributed by Thierry Arnoux,
+       28-Apr-2025.) $)
+    df-erl $a |- ~RL = ( r e. _V , s e. _V |-> [_ ( .r ` r ) / x ]_
+          [_ ( ( Base ` r ) X. s ) / w ]_
+       { <. a , b >. | ( ( a e. w /\ b e. w ) /\ E. t e. s
+         ( t x ( ( ( 1st ` a ) x ( 2nd ` b ) ) ( -g ` r ) ( ( 1st ` b )
+                                    x ( 2nd ` a ) ) ) ) = ( 0g ` r ) ) } ) $.
+  $}
+
+  ${
+    $d r s x w a b k $.
+    $( Define the operation giving the localization of a ring by a given set.
```

Comment:
Maybe add more details here as well? Something like
```
Define the operation giving the localization of a ring ` r ` by a given set ` s `.
The localized ring ` r RLocal s ` is the set of equivalence classes of ` r `
over the relation ` r ~RL s ` with addition and multiplication defined naturally.
```

Reply:
I'm just adding that this is the set of equivalence classes of _pairs_ of elements of ` r ` over the relation ` r ~RL s `.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51